cmd/kubeadm/app/util/system/cgroup_validator.go

```diff
@@ -41,15 +41,15 @@ const (
 )
 
 // Validate is part of the system.Validator interface.
-func (c *CgroupsValidator) Validate(spec SysSpec) (error, error) {
+func (c *CgroupsValidator) Validate(spec SysSpec) ([]error, []error) {
 	subsystems, err := c.getCgroupSubsystems()
 	if err != nil {
-		return nil, errors.Wrap(err, "failed to get cgroup subsystems")
+		return nil, []error{errors.Wrap(err, "failed to get cgroup subsystems")}
 	}
 	return nil, c.validateCgroupSubsystems(spec.Cgroups, subsystems)
 }
 
-func (c *CgroupsValidator) validateCgroupSubsystems(cgroupSpec, subsystems []string) error {
+func (c *CgroupsValidator) validateCgroupSubsystems(cgroupSpec, subsystems []string) []error {
```

Comment:
No need to change this one. It can return only one error and no warnings.

Reply:
you are correct. my apologies, I will update!